oida/utils.py

```diff
@@ -15,3 +16,20 @@ def run_black(value: str, *, filename: Path | None = None) -> str:
     process = subprocess.run(command, capture_output=True, encoding="utf-8")
 
     return process.stdout if process.returncode == 0 else value
+
+
+def path_in_glob_list(path: str, glob_list: list[str]) -> bool:
+    """
+    Returns True if the path is included in the provided glob list.
+    """
+
+    path_list = (path or "").split(".")
```

Comment:
```suggestion
    path_list = path.split(".")
```

Shouldn't be needed when this is always a string